packages/colours/Colours.md

```diff
@@ -18,124 +18,171 @@ For more information on the usage of each colour please refer to the [colour pri
+          <Paragraph>
+            <Strong>Lavender Blush</Strong>
+          </Paragraph>
+          <Paragraph>colorLavenderBlush</Paragraph>
           <Paragraph>Hex: #FFF6F8</Paragraph>
         </Box>
       </Box>
     </FlexGrid.Col>
     <FlexGrid.Col xs={4}>
       <Box vertical={3}>
-        <span className="docs_color-preview" style={{backgroundColor: 'rgb(242,239,244)'}}></span>
+        <span className="docs_color-preview" style={{ backgroundColor: 'rgb(242,239,244)' }} />
         <Box between={3}>
-          <Paragraph><Strong>White Lilac</Strong></Paragraph>
-          <Paragraph>$color-white-lilac</Paragraph>
+          <Paragraph>
+            <Strong>White Lilac</Strong>
+          </Paragraph>
+          <Paragraph>colorWhiteLilac</Paragraph>
           <Paragraph>Hex: #F2EFF4</Paragraph>
         </Box>
       </Box>
     </FlexGrid.Col>
     <FlexGrid.Col xs={4}>
       <Box vertical={3}>
-        <span className="docs_color-preview" style={{backgroundColor: 'rgb(216,216,216)'}}></span>
+        <span className="docs_color-preview" style={{ backgroundColor: 'rgb(216,216,216)' }} />
         <Box between={3}>
-          <Paragraph><Strong>Gainsboro</Strong></Paragraph>
-          <Paragraph>$color-gainsboro</Paragraph>
+          <Paragraph>
+            <Strong>Gainsboro</Strong>
+          </Paragraph>
+          <Paragraph>colorGainsboro</Paragraph>
           <Paragraph>Hex: #D8D8D8</Paragraph>
         </Box>
       </Box>
     </FlexGrid.Col>
     <FlexGrid.Col xs={4}>
       <Box vertical={3}>
-        <span className="docs_color-preview" style={{backgroundColor: 'rgb(247,247,248)'}}></span>
+        <span className="docs_color-preview" style={{ backgroundColor: 'rgb(247,247,248)' }} />
         <Box between={3}>
-          <Paragraph><Strong>Athens Grey</Strong></Paragraph>
-          <Paragraph>$color-athens-grey</Paragraph>
+          <Paragraph>
+            <Strong>Athens Grey</Strong>
+          </Paragraph>
+          <Paragraph>colorAthensGrey</Paragraph>
           <Paragraph>Hex: #F7F7F8</Paragraph>
         </Box>
       </Box>
     </FlexGrid.Col>
     <FlexGrid.Col xs={4}>
       <Box vertical={3}>
-        <span className="docs_color-preview" style={{backgroundColor: 'rgb(244,249,242)'}}></span>
+        <span className="docs_color-preview" style={{ backgroundColor: 'rgb(244,249,242)' }} />
         <Box between={3}>
-          <Paragraph><Strong>Panache</Strong></Paragraph>
-          <Paragraph>$color-panache</Paragraph>
+          <Paragraph>
+            <Strong>Panache</Strong>
+          </Paragraph>
+          <Paragraph>colorPanache</Paragraph>
           <Paragraph>Hex: #F4F9F2</Paragraph>
         </Box>
       </Box>
     </FlexGrid.Col>
   </FlexGrid.Row>
 </FlexGrid>
 ```
+
+### SASS
+
+```scss static
+@import '~@tds/core-colours/colours';
+```
+
+SASS variables can continue to be used, though they will be **deprecated** at some point. See the table below for the corresponing SASS variable.
```

Comment:
You can remove the "SASS variables can continue to be used...at some point." sentence.

Please change some of the wording in the next sentence:

> See the table below for the `full list of JS variables and their corresponding` SASS variables.